runtime/doc/news.txt

```diff
@@ -230,6 +230,10 @@ The following changes to existing APIs or features add new behavior.
   In addition, |nvim_buf_get_extmarks()| has gained an "overlap" option to
   return such ranges even if they started before the specified position.
 
+• |:range!| now pipe by default to |'shell'| on startup if a {filter} external
+  program is not given. But if given, the latter is remembered.
```

Comment:
This is confusing to mention in the news since it's not net. Users can find the details by visiting the docs.

```suggestion
  program is not given.
```